### PR TITLE
Added Japanese Translation

### PR DESCRIPTION
**Pull request by flojurian**
Hi everyone,

I would like to use the Berlin Code of Conduct for my soon to kickoff Clojure meetups in Fukuoka, Japan. The participants should be able to understand the English version, but some will surely be happy if they can also read it in Japanese. I therefore prepared a translation together with my wife. I could put this online on my own site, but I would like to offer you this addition for the 'official' homepage first. We'd be happy if you'd accept this contribution.

Friendly regards from Japan & Schoenes Wochenende!
  Florian

P.S.: While adding the actual translation I took the liberty to fix some typos and extend the README a little bit, I hope that's OK.

### CODE REVIEW

**Comment by hagenburger** [comment]
@flojurian thanks a lot for your effort! [I’ve asked via Twitter](https://twitter.com/hagenburger/status/1205408888364916736) for a reviewer. 

If you like to have your user group online ASAP, feel free to open a separate pull request for it. If possible, rebase to `master`.

File: locales/ja.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+ja:
+  header:
+    title: "ベルリン行動規範"
+    sub-title: "すべての会議およびユーザーグループのための行動規範"
+  supporters:
+    title: "本行動規範の支持者"
+    sub-title: "ベルリン行行動規範を支持する会議およびユーザーグループ："
```
Comment (morizyun) [inline locales/ja.yml]: Is `行行` `行` ?

File: source/localizable/_code-of-conduct.ja.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+オープン［ソース／文化／技術］の市民権 
+--------------------------------------
+
+本行動規範の補足的な目標は、私たちの行動とそれが及ぼす私たちのコミュニティへの影響との関係を参加者に認識させることにより、オープン［ソース／文化／技術］を広めることです。 
+
+コミュニティはそれらが存在する社会を反映しており、社会に存在する様々な形態の不平等と権力濫用に対抗するには積極的な行動が不可欠です。 
+
+私たちのコミュニティが歓迎し友好的になるためにより一層努力し、すべての参加者に最大限に貢献するよう促している人がいたらお知らせ下さい。
```
Comment (morizyun) [inline source/localizable/_code-of-conduct.ja.md]: IMO 

```
私たちのコミュニティが歓迎され、友好的であり、すべての参加者が最大限に貢献することを奨励するために 必要以上の努力をしている人を見かけたら、知らせてください。
```

It sounds more natural.

**Comment by morizyun** [comment]
I think it is excellent!!

File: locales/ja.yml (hunk 1 above)
Comment (flojurian) [reply locales/ja.yml]: It surely is, thanks. Fixed the typo.

File: source/localizable/_code-of-conduct.ja.md (hunk 2 above)
Comment (flojurian) [reply source/localizable/_code-of-conduct.ja.md]: コミュニティが歓迎され = community is welcomed
のコミュニティが歓迎し = community is welcoming

奨励するために = to encourage
促している人 = someone who encourages

でしょう？ 
We'd like to stick with the current version.

**Comment by flojurian** [comment]
> @flojurian thanks a lot for your effort! [I’ve asked via Twitter](https://twitter.com/hagenburger/status/1205408888364916736) for a reviewer.
> 
> If you like to have your user group online ASAP, feel free to open a separate pull request for it. If possible, rebase to `master`.

Thanks but there's no need to hurry. I'll just keep updating while there are change requests.

File: source/localizable/_code-of-conduct.ja.md (hunk 2 above)
Comment (morizyun) [reply source/localizable/_code-of-conduct.ja.md]: コミュニティが歓迎され = community is welcomed
のコミュニティが歓迎し = community is welcoming

It is right. so you can keep the current version

---

奨励するために = to encourage
促している人 = someone who encourages

Unfortunately, I could not under stand the following current version.

```
すべての参加者に最大限に貢献するよう促している人がいたらお知らせ下さい。 
```

Comment (morizyun) [reply source/localizable/_code-of-conduct.ja.md]: Sorry, please foget my text.

**Comment by hagenburger** [comment]
@flojurian thanks again for your work and @morizyun and @JuanitoFatas for reviewing!

@flojurian can you rebase your branch to the latest `master` or should I do it? When this is done, I’m happy to merge.

**Comment by flojurian** [comment]
@hagenburger Nico, please feel free to merge this as you like and thanks for accepting our translation!

@morizyun and @JuanitoFatas
レビューをありがとうございました、これからも宜しくお願いします！

**Comment by flojurian** [comment]
@hagenburger Dear Nico, if your (or someone elses) schedule allows it, could you merge this until the end of next week maybe? That would be absolutely fantastic because I'll have the first meetup session coming Saturday. Thanks a lot and sorry for the trouble.

**Comment by chastell** [comment]
@flojurian Merged and deployed, thanks so much for this translation! 🙌

**Comment by flojurian** [comment]
@chastell Dear Piotr, thank you for your time! You're very welcome.

**Pull request merged**